spec/unit/run.spec.js

```diff
@@ -201,14 +201,7 @@ describe('run', () => {
     describe('help', () => {
         it('should print out usage and help', () => {
             spyOn(console, 'log');
-
-            // Rewiring the process object in entirety does not work on NodeJS 12.
-            // Rewiring members of process however does work
-            // https://github.com/apache/cordova-android/issues/768
-            // https://github.com/jhnns/rewire/issues/167
-            run.__set__('process.exit', _ => null);
-            run.__set__('process.cwd', _ => '');
-            run.__set__('process.argv', ['', '']);
+            spyOn(process, 'exit');
```

Comment:
```suggestion
            // blocks the process.exit() call from going through
            // (in this test case only)
            spyOn(process, 'exit');
```

Reply:
I get your surprise. I've been there myself. However, there are at least 170 instances of `spyOn` calls that do not specify a spy strategy (and thus are stubs) in the Cordova projects I have checked out locally, alone. I really don't want to start commenting them all.

Reply:
OK, merge it then.